pkg/volume/cinder/volumeservice/connection.go

```diff
@@ -88,10 +88,10 @@ func getKeystoneVolumeService(cfg cinderConfig) (*gophercloud.ServiceClient, err
 		return nil, err
 	}
 
-	volumeService, err := openstack.NewBlockStorageV2(provider,
```

Comment:
standalone cinder is deprecated, no need to update here

Reply:
I prefer to keep the consistency within all packages.